src/Orleans/Runtime/RequestContextUtil.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+
+namespace Orleans.Runtime
+{
+    public class RequestContextUtil
```

Comment:
@sergeybykov that's right, so it needs a reference to SerializationManager. Currently the class is just a stub